core/presentation/designsystem/src/main/java/com/nolawiworkineh/designsystem/components/TaskyScaffold.kt

```diff
@@ -0,0 +1,45 @@
+package com.nolawiworkineh.designsystem.components
+
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.FabPosition
+import androidx.compose.material3.HorizontalDivider
+import androidx.compose.material3.Scaffold
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import com.nolawiworkineh.designsystem.Theme.TaskyBlack
+import com.nolawiworkineh.designsystem.Theme.TaskyGray
+
+@Composable
+fun TaskyScaffold(
+    modifier: Modifier = Modifier,
+    isStartFab: Boolean = false,
```

Comment:
Minor - this property is a bit unclear in what I behaviour I can expect when I use this Composable. Instead, I would directly use the `FabPosition` from the `Scaffold` Composable:

```
@Composable
fun TaskyScaffold(
  modifier: Modifier = Modifier,
  fabPosition: FabPosition = FabPosition.Start
) { }
```

Reply:
I did this because tasky has 2 different action buttons one at the start(in the register screen) and the other at the end (home screen) so I wanted to do something that would be dynamic. So if isFabStart is true, then the FAB would be at the start, if not at the default position which is at the end